internal/gatewayapi/listener.go

```diff
@@ -301,6 +302,16 @@ func (t *Translator) processAccessLog(envoyproxy *egv1a1.EnvoyProxy, resources *
 				irAccessLog.OpenTelemetry = append(irAccessLog.OpenTelemetry, al)
 			}
 		}
+
+		var validExprs []string
+		for _, expr := range accessLog.Matches {
+			if !validCELExpression(expr) {
+				t.Logger.Info("invalid CEL expression", "expression", expr)
```

Comment:
can we collect these `errs` and update status without returning early (so we allow the good matches to be translated) ?